core/src/test/resources/xform_tests/test_display_conditions_regression.xml

```diff
@@ -0,0 +1,46 @@
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml"
+    xmlns:jr="http://openrosa.org/javarosa">
+    <h:head>
+        <h:title>Label Display Conditions Test</h:title>
+        <model>
+            <instance>
+                <data
+                    name="display conditions"
+                    xmlns="http://openrosa.org/test/actions"
+                    uiVersion="1"
+                    version="1">
+                    <threshold>0</threshold>
+                    <labels>
+                        <label_should_show />
+                        <label_should_not_show />
+                    </labels>
+                </data>
+            </instance>
+
+            <bind nodeset="/data/labels/label_should_show" relevant="/data/threshold = 0" />
+            <bind nodeset="/data/labels/label_should_not_show" relevant="/data/threshold &lt; 0" />
```

Comment:
@ctsims The test will fail with the relevant expression here, but will pass if you change it to `relevant="/data/threshold &lt; -1"`